src/plugins/intel_cpu/src/dnnl_scratch_pad.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include "common/memory.hpp"
+#include "cpu_memory.h"
+#include "dnnl_extension_utils.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class DnnlScratchPad {
+    DnnlMemoryMngrPtr mgrPtr;
+    dnnl::engine eng;
+
+public:
+    DnnlScratchPad(dnnl::engine eng) : eng(eng) {
+        mgrPtr = std::make_shared<DnnlMemoryMngr>(std::unique_ptr<MemoryMngrWithReuse>(new MemoryMngrWithReuse()));
+    }
+
+    MemoryPtr getScratchPadMem(const DnnlMemoryDescPtr& md) {
```

Comment:
```suggestion
    MemoryPtr getScratchPadMem(const MemoryDescPtr& md) {
```
That would allow to create memory from any descriptor.

Reply:
Done